src/net_processing.cpp

```diff
@@ -1758,7 +1756,7 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
             }
 
             // Get recent addresses
-            if (pfrom->fOneShot || pfrom->nVersion >= CADDR_TIME_VERSION || connman->GetAddressCount() < 1000)
+            if (pfrom->fOneShot || connman->GetAddressCount() < 1000)
```

Comment:
This looks wrong? Shouldn't it be unconditional now, since the condition was `||`?

Reply:
Yep, def wrong. Thanks for catching that.